paws.common/R/config.R

```diff
@@ -95,6 +98,41 @@ check_config_file_region <- function() {
   return(region)
 }
 
+# Tries to get the region from the config file
+check_config_file_assume_role <- function() {
```

Comment:
Probably should check whether it got a complete set of creds before returning.

Reply:
addressed